source/manual/setting-up-new-rails-app.html.md

```diff
@@ -172,88 +208,94 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 ```
 
-</details>
-
 ## Replace the default README.md
 
-We have a common structure that is used for GOV.UK apps. Fill in some basic
-details to get started with your app and flesh it out further as your project
-develops.
+Replace the default `README.md` file with the following content structure:
 
 ```markdown
 # App Name
 
-One paragraph description and purpose.
+Information about the app's description and purpose.
 
-## Screenshots (if there's a client-facing aspect of it)
+## Technical documentation
 
-## Live examples (if available)
+Write a single paragraph including a general technical overview of the app.
 
-- [gov.uk/thing](https://www.gov.uk/thing)
+### Testing
 
-## Nomenclature
+Information about how to test the app.
 
-- **Word**: definition of word, and how it's used in the code
+## Further documentation
 
-## Technical documentation
+More information on how the app works.
 
-Write a single paragraph including a general technical overview of the app.
+## API documentation
 
-### Dependencies
+If applicable, include API reference information including the following sections:
 
-- [dependency]() - purpose
+- resources
+- endpoints and methods
+- parameters
+- example requests and responses
+- error codes
 
-### Running the application
+## Licence
 
-How to run the app
+Link to the [MIT License][LICENCE].
+```
 
-### Running the test suite
+See the example [account-api README.md][https://github.com/alphagov/account-api/blob/main/README.md] for more information.
```

Comment:
Sorry, I should have been clearer - it's only when using a reference that the syntax is `[title][reference]`.  When using a URL, it's still `[title](url)`.